test/verify/check-packagekit

```diff
@@ -54,9 +54,15 @@ class TestUpdates(MachineCase):
         m.start_cockpit()
         b.login_and_go("/updates")
 
+        # PK starts from a blank state, thus should force refresh
+        b.wait_in_text("#state", "Refreshing")
```

Comment:
That's right. I haven't seen it fail in practice, but nominally it's a transient state, and there isn't much leverage in artificially making that take longer. The test could probably add 100 local test packages instead of just two for that if it becomes a problem.

The alternative would be to not cover this transient state at all (can do if you prefer).